FF132 Relnote fetchPriority supported (#36259)
mdn/content: files/en-us/mozilla/firefox/releases/132/index.md

### APIs

> - Developers can now provide a hint about the relative priority for fetching a particular resources compared to other resources of the same type.

Reviewer: " a particular resources" > I guess singular?

Author: Thanks @fred-wang - you'd think I didn't know how to write. Doh.

```suggestion
- Developers can now provide a hint about the relative priority for fetching a particular resource compared to other resources of the same type.
```

>   This can be used, for example, to indicate that a image is more or less important to the user experience that other images. The priority may be set using the `fetchpriority` attribute of the {{htmlelement("link")}}, {{htmlelement("script")}}, or {{htmlelement("img")}} elements, the `fetchPriority` property of the {{domxref("HTMLLinkElement")}}, {{domxref("HTMLScriptElement")}}, or {{domxref("HTMLImageElement")}} interfaces, the [`options.priority`](/en-US/docs/Web/API/RequestInit#priority) parameter passed to the [`Request()` constructor](/en-US/docs/Web/API/Request/Request), or in the HTTP {{httpheader("Link")}} header. Note that this mechanism is complementary to other ways of setting the priority, such as preloading. ([Firefox bug 1854077](https://bugzil.la/1854077)).

Reviewer: an* image

Reviewer: s/that/than/

### APIs

> - The `fetchpriority` attribute of the {{htmlelement("link")}}, {{htmlelement("script")}}, and {{htmlelement("img")}} elements, the `fetchPriority` property of the {{domxref("HTMLLinkElement")}}, {{domxref("HTMLScriptElement")}}, and {{domxref("HTMLImageElement")}} interfaces, the [`options.priority`](/en-US/docs/Web/API/RequestInit#priority) parameter passed to the [`Request()` constructor](/en-US/docs/Web/API/Request/Request), and the `fetchpriority` directive in the HTTP {{httpheader("Link")}} header, are now supported. 
>   These allow developers to provide a hint about the relative priority for fetching a particular resource compared to other resources of the same type, and can be used alongside other ways of setting the priority, such as preloading. ([Firefox bug 1854077](https://bugzil.la/1854077)).

Author: ```suggestion
- The `fetchpriority` attribute of the {{htmlelement("link")}}, {{htmlelement("script")}}, and {{htmlelement("img")}} elements, the `fetchPriority` property of the {{domxref("HTMLLinkElement")}}, {{domxref("HTMLScriptElement")}}, and {{domxref("HTMLImageElement")}} interfaces, the [`options.priority`](/en-US/docs/Web/API/RequestInit#priority) parameter passed to the [`Request()` constructor](/en-US/docs/Web/API/Request/Request), and the `fetchpriority` directive in the HTTP {{httpheader("Link")}} header, are now supported. These allow developers to provide a hint about the relative priority for fetching a particular resource compared to other resources of the same type, and can be used alongside other ways of setting the priority, such as preloading. ([Firefox bug 1854077](https://bugzil.la/1854077)).
```

> - Developers can now provide a hint about the relative priority for fetching a particular resource compared to other resources of the same type.
>   This can be used, for example, to indicate that an image is more or less important to the user experience than other images. The priority may be set using the `fetchpriority` attribute of the {{htmlelement("link")}}, {{htmlelement("script")}}, or {{htmlelement("img")}} elements, the `fetchPriority` property of the {{domxref("HTMLLinkElement")}}, {{domxref("HTMLScriptElement")}}, or {{domxref("HTMLImageElement")}} interfaces, the [`options.priority`](/en-US/docs/Web/API/RequestInit#priority) parameter passed to the [`Request()` constructor](/en-US/docs/Web/API/Request/Request), or in the HTTP {{httpheader("Link")}} header. Note that this mechanism is complementary to other ways of setting the priority, such as preloading. ([Firefox bug 1854077](https://bugzil.la/1854077)).

Author: Thanks @pepelsbey . I prefer the flow of my original text because it is a big list of things that might not be relevant to the reader. But I don't care strongly, so here we go.

```suggestion
- The `fetchpriority` attribute of the {{htmlelement("link")}}, {{htmlelement("script")}}, and {{htmlelement("img")}} elements, the `fetchPriority` property of the {{domxref("HTMLLinkElement")}}, {{domxref("HTMLScriptElement")}}, and {{domxref("HTMLImageElement")}} interfaces, the [`options.priority`](/en-US/docs/Web/API/RequestInit#priority) parameter passed to the [`Request()` constructor](/en-US/docs/Web/API/Request/Request), and the `fetchpriority` directive in the HTTP {{httpheader("Link")}} header, are now supported. 
  These allow developers to provide a hint about the relative priority for fetching a particular resource compared to other resources of the same type, and can be used alongside other ways of setting the priority, such as preloading. ([Firefox bug 1854077](https://bugzil.la/1854077)).
```
